specification/confidentialledger/data-plane/Microsoft.ConfidentialLedger/preview/2024-12-09-preview/confidentialledger.json

```diff
@@ -2094,6 +2100,14 @@
       "description": "The collection id.",
       "type": "string"
     },
+    "TagIdParameter": {
+      "name": "tags",
```

Comment:
hey Musa, in the current implementation, the GET endpoint uses `tag` singular as the `TagIdParameter`, while the POST endpoint uses `tags`.  

For retrieving tag values, we have currently decided to not merge the results of multiple tags, we can discuss if this makes sense, or if we want to implement it.